cli/cli/commands/cluster/set/set.go

```diff
@@ -60,51 +62,62 @@ func run(ctx context.Context, flags *flags.ParsedFlags, args *args.ParsedArgs) e
 		return nil
 	}
 
-	engineManagerOldCluster, err := engine_manager.NewEngineManager(ctx)
-	if err != nil {
-		return stacktrace.Propagate(err, "An error occurred creating an engine manager.")
+	// at the moment docker clusters are local only, we stop engine in them in favor of switching to any other cluster
```

Comment:
Should we fail if the current context is a remote one so we don't end up stopping the engine on the remote docker instance?  The user would have to first switch to the local context using `kurtosis context switch default` before setting the cluster to kubernetes.

Reply:
in my mind i had isolated the context & the cluster users, I will do a check for current context type and pop an error with the recommended step if so